sdcm/fill_db_data.py

```diff
@@ -3343,7 +3343,10 @@ def prepare_keyspaces_and_tables(self):
         with self.db_cluster.cql_connection_patient(node) as session:
             # override driver consistency level
             session.default_consistency_level = ConsistencyLevel.QUORUM
-
+            session.execute("""
```

Comment:
this is kind of not related to this whole file...
scylla-bench isn't used here at all...

Anyhow, let's leave it like that

Reply:
my question is, why is it now happening, and why in this file, and not in `upgrade_test.py`?
also, it should be used only when we do have a `scylla-bench` command to be ran, no? in that case, it could be inside the `run_stress` code, or not?

Reply:
it's basically  a patch, until s-b would be fixed, no need to make it more complex that.
@vponomaryov seem to be looking into it.